### PR TITLE
Make use of emplace in nonassociative containers.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by donaloconnor** [comment]
utACK

I had a look through all these changes and couldn't see any of the potential edge cases where this can lead to a change of behaviour (Eg: calling an explicit ctor instead of an implicit as push_back does).

File: src/bitcoin-tx.cpp [hunk 1]
```diff
@@ -546,7 +546,7 @@ static void MutateTxSign(CMutableTransaction& tx, const std::string& flagStr)
             throw std::runtime_error("unknown sighash flag/sign option");
 
     std::vector<CTransaction> txVariants;
```
Comment (eklitzke) [inline src/bitcoin-tx.cpp]: why not 
```
std::vector<CTransactoin> txVariants{tx};
```

File: src/qt/rpcconsole.cpp [hunk 2]
```diff
@@ -147,7 +147,7 @@ class QtRPCTimerInterface: public RPCTimerInterface
 bool RPCConsole::RPCParseCommandLine(std::string &strResult, const std::string &strCommand, const bool fExecute, std::string * const pstrFilteredOut)
 {
     std::vector< std::vector<std::string> > stack;
```
Comment (eklitzke) [inline src/qt/rpcconsole.cpp]: same, just initialize the data element in the ctor

**Comment by laanwj** [comment]
Going to close this as it seems to be inactive.

**Pull request closed without merging**